agents/sbd/fence_sbd.py

```diff
@@ -357,6 +360,22 @@ def define_new_opts():
         "order": 200
         }
 
+    all_opt["crashdump"] = {
+        "getopt" : "",
+        "longopt" : "crashdump",
+        "help" : "--crashdump=[0|1]              Enable crashdump (Default: 0, disabled)",
```

Comment:
You can remove `=[0|1]` here, so it only requires `--crashdump` to enable it from CLI, and from Pacemaker or stdin it will accept `crashdump=[1|yes|on|true]` to enable it.

Also run it with -h check if  the description lines up with the other parameters.